components/MdInput.tsx

```diff
@@ -60,7 +69,26 @@ export const MdInput: React.FC<MdInputProps> = ({
       <Button color={previewBtnColor} onClick={() => setPreview(true)}>
         Preview
       </Button>
-      {displayOption}
+      {preview && (
+        <Markdown data-testid="markdown" className={`${styles['preview']}`}>
+          {value}
+        </Markdown>
+      )}
+      <textarea
+        onMouseDown={() => {
+          mouseDownHeightRef.current = textareaRef.current!.clientHeight
+        }}
+        ref={textareaRef}
+        value={value}
+        className={`${styles['textarea']}${preview ? ' d-none' : ''}`}
+        onChange={e => {
+          if (!height && textareaRef.current) autoSize(textareaRef.current)
+          onChange(e.target.value)
+        }}
+        placeholder={placeHolder}
+        style={height ? { height: height + 'px' } : undefined}
+        data-testid="textbox"
```

Comment:
Since you are no longer using this testid and getting the element by role in the test, you can remove this.

Reply:
I believe there are some other test files that are using the test Id. I should be able to update them as well tho, I will check tomorrow

Reply:
Ah, if this component/testid is being used be other tests then sure can let it remain.

Reply:
Currently AdminLessonChallenges.test.js , FormCard.test.js, CommentBox.test.js and ChallengeMaterial.test.js are using the ID. I will clean that up on the next PR when I add the markdown-toolbar buttons